packages/ketcher-react/src/script/ui/state/hotkeys.ts

```diff
@@ -119,6 +120,11 @@ function keyHandle(dispatch, state, hotKeys, event) {
           dispatch
         })
       } else {
+        if (newAction.tool === 'select') {
+          const savedSelectionTool = SettingsManager.getSettings().selectionTool
```

Comment:
Could we avoid creating an extra variable and write just:
`newAction = SettingsManager.getSettings().selectionTool`?